python/nmodl/__init__.py

```diff
@@ -33,8 +35,7 @@
 
 try:
     # Try importing but catch exception in case bindings are not available
-    from ._nmodl import NmodlDriver, to_json, to_nmodl  # noqa
-    from ._nmodl import __version__
+    from ._nmodl import NmodlDriver, to_json, to_nmodl, __version__  # noqa
```

Comment:
Why is this happening in a `try` block? What's the difference between a Python package `nmodl` with python binding and without? Could we require the "bindings" to always be present and simply error out?

Reply:
Apparently removing it breaks this CI: https://github.com/BlueBrain/nmodl/actions/runs/8138354896/job/22238986595?pr=1147

Reply:
Okay, might be a bug in the tests.

Reply:
I've reverted back the commit; if it's a bug in the tests, I think it's best we investigate in a separate issue/PR.